src/GalleryApp.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Header from './Components/Header';
+import Footer from './Components/Footer';
+
+class GalleryApp extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: null,
+      isLoaded: false,
+      items: []
+    }
+  }
+
+  componentDidMount() {
+    fetch("https://jsonplaceholder.typicode.com/photos")
+    .then(res => res.json())
+    .then(
+      (result) => {
+        this.setState({
+          isLoaded: true,
+          items: result
+        });
+      },
+      (error) => {
+        this.setState({
+          isLoaded: true,
+          error
+        });
+      }
+    )
+  }
+
+  render() {
+    const { error, isLoaded, items } = this.state;
+
+    if (error) {
+      return <div>Error: {error.message}</div>;
+    } else if (!isLoaded) {
```

Comment:
This doesn't need to be an if else. The return stops the flow so each one can be just an if and the last one (the actually component can be outside any if.

Reply:
I understand now. Iv'e completed the commits from your updates.